### PR TITLE
blog: April newsletter

### PR DESCRIPTION
**Pull request by GregSutcliffe**
@mmoll do you want me to mention Stretch is ready for Nightly in here, or leave it a bit?

### CODE REVIEW

**Comment by GregSutcliffe** [comment]
@mmoll added a paragraph as per IRC chat

File: _posts/2017-04-30-foreman-community-newsletter---april-2017.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+Release Candidates are also available. See the [release
+announcement][k3_4_rc2]. As ever, we value the testing, and if you hit any
+issues, please to raise a bug on the [Katello tracker][k_issues].
+
+### Debian 9 (Stretch) support lands in nightly
+
+Thanks to the hard work of the packaging team (mostly @mmoll!) packages for
+Debian 9 (Stretch) are now available in nightly, and feedback on how well they
+work for people is welcome. If you do test, please be aware that MySQL support
+is currently not functional due to an issue in the `puppetlabs-mysql` module,
```
Comment (mmoll) [inline _posts/2017-04-30-foreman-community-newsletter---april-2017.md]: It's not really an "issue", just a new release of the module would be required.

Comment (GregSutcliffe) [reply _posts/2017-04-30-foreman-community-newsletter---april-2017.md]: Fixed

File: _posts/2017-04-30-foreman-community-newsletter---april-2017.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+users, and much more) then jump in and get involved with testing the RCs.
+
+As ever, we rely on you guys to test the release candidates (and *so* many of
+you do, thanks!), and again we're seeing that participation bearing fruit.
+Please do continue to log the [bugs][issues] you find in the RCs so that we can
+ensure the 1.15 release is as smooth as possible.
+
+### 1.14.3 Bugfix release
+
+Foreman 1.14.3 bugfix release was published on [April 4th][1_14_2], containing
```
Comment (mmoll) [inline _posts/2017-04-30-foreman-community-newsletter---april-2017.md]: the link must be 1_14_3

Comment (mmoll) [inline _posts/2017-04-30-foreman-community-newsletter---april-2017.md]: the link must be 1_14_3

**Review by mmoll — DISMISSED** [review]

**Comment by mmoll** [comment]
fixed the 1.14.3 link and merged, thanks @GregSutcliffe!

**Pull request merged**